src/main/java/domain/KontoUser.kt

```diff
@@ -1,15 +1,15 @@
 package domain
 
+import java.util.*
 import javax.persistence.*
 
 @Entity
-@Table(name = "KontoUser")
+@Table(name = "KONTOUSER")
```

Comment:
for SQL it's default to capitalize keywords as `WHERE, LIKE and SELECT` and keep table names to lowercase

Reply:
DONE